deps/mbedtls.mk

```diff
@@ -7,12 +7,8 @@ else
 endif
 MBEDTLS_URL = https://tls.mbed.org/download/$(MBEDTLS_SRC).tgz
 
-MBEDTLS_OBJ_SOURCE := $(BUILDDIR)/mbedtls-$(MBEDTLS_VER)/library/libmbedtls.$(SHLIB_EXT) \
-			$(BUILDDIR)/mbedtls-$(MBEDTLS_VER)/library/libmbedx509.$(SHLIB_EXT) \
-			$(BUILDDIR)/mbedtls-$(MBEDTLS_VER)/library/libmbedcrypto.$(SHLIB_EXT)
-MBEDTLS_OBJ_TARGET := $(build_shlibdir)/libmbedtls.$(SHLIB_EXT) \
-			$(build_shlibdir)/libmbedx509.$(SHLIB_EXT) \
-			$(build_shlibdir)/libmbedcrypto.$(SHLIB_EXT)
+MBEDTLS_OBJ_SOURCE := $(BUILDDIR)/mbedtls-$(MBEDTLS_VER)/library/libmbedcrypto.$(SHLIB_EXT)
+MBEDTLS_OBJ_TARGET := $(build_shlibdir)/libmbedcrypto.$(SHLIB_EXT)
```

Comment:
I left only one, because if I put three and use this variable as dependency in other Makefiles (libssh2) then make builds it 3 times.